index.html

```diff
@@ -998,21 +998,37 @@
       function createTableHead(headings) {
         const tr = $('<tr></tr>')
         tr.append($('<th class="text-center">#</th>'))
-        for (let h=0; h<headings.length; h++) {
-          tr.append($(`<th>${headings[h]}</th>`))
+        if (headings.length == 0) {
```

Comment:
The `headings` for `listClusters` does have one element, `"List"`.

Also check `CodeDeploy.listOnPremisesInstances` line 443 to see how this can be done without special exceptions in code.

Reply:
Basically, I can check if the head is empty, just wanted to put some meaningful title.
I'll change it.